CHANGELOG.md

```diff
@@ -1,7 +1,11 @@
 x.y.z Release notes (yyyy-MM-dd)
 =============================================================
+### Breaking Changes
+* `Credentials(username:password:)` is renamed to `Credentials(email:password:)`
+
 ### Enhancements
-* None.
+* Add `User.loggedIn`.
+* Refactor `Credentials` and `User` methods for MongoDB Realm consistency.
```

Comment:
This isn't an accurate or useful changelog message. There isn't any refactoring done in this PR (it's all just renaming things), and if there was that wouldn't normally get a changelog entry because it isn't relevant to users of the SDK.